app/Models/Bible/Bible.php

```diff
@@ -330,6 +330,8 @@ public function scopeWithRequiredFilesets($query, $type_filters)
             if ($type_filters['media']) {
                 $q->where('bible_filesets.set_type_code', $type_filters['media']);
             }
+
+            $q->select(\DB::raw(1));
```

Comment:
I'm not sure I understand this

Reply:
It is related to avoid to select all columns inside a exist operator because we don't need all columns.